### PR TITLE
Change data generating in classification, fbeta, mae

### PR DESCRIPTION
**Pull request by puhuk**
Description: Change data generating in `classification_report`, `fbeta`, `mae`

Check list:

- [x] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: tests/ignite/metrics/test_classification_report.py [hunk 1]
```diff
@@ -77,24 +80,22 @@ def update(engine, i):
...
-        y_true = torch.randint(0, 2, size=(offset * idist.get_world_size(), n_classes, 6, 8)).to(device)
-        y_preds = torch.randint(0, 2, size=(offset * idist.get_world_size(), n_classes, 6, 8)).to(device)
+        y_true = torch.randint(0, 2, size=(n_iters * batch_size, n_classes, 6, 8)).to(device)
+        y_preds = torch.randint(0, 2, size=(n_iters * batch_size, n_classes, 6, 8)).to(device)
 
         def update(engine, i):
             return (
-                y_preds[i * s + rank * offset : (i + 1) * s + rank * offset, ...],
-                y_true[i * s + rank * offset : (i + 1) * s + rank * offset, ...],
+                y_preds[i * batch_size : (i + 1) * batch_size],
```
Comment (vfdev-5) [inline tests/ignite/metrics/test_classification_report.py:97]: @puhuk why do you suppress `...` here ?

File: tests/ignite/metrics/test_fbeta.py [hunk 2]
```diff
@@ -91,21 +91,21 @@ def update_fn(engine, batch):
...
+        torch.manual_seed(12 + rank)
+
+        y_true = torch.randint(0, n_classes, size=(n_iters * batch_size,)).to(device)
+        y_preds = torch.rand(n_iters * batch_size, n_classes).to(device)
 
         def update(engine, i):
             return (
-                y_preds[i * s + rank * offset : (i + 1) * s + rank * offset, :],
-                y_true[i * s + rank * offset : (i + 1) * s + rank * offset],
+                y_preds[i * batch_size : (i + 1) * batch_size],
```
Comment (vfdev-5) [inline tests/ignite/metrics/test_fbeta.py:107]: Same here, missing `:`

File: tests/ignite/metrics/test_classification_report.py (hunk 1 above)
Comment (puhuk) [reply tests/ignite/metrics/test_classification_report.py:97]: I thought the output is same. Let me check and correct this.

File: tests/ignite/metrics/test_classification_report.py [hunk 3]
```diff
@@ -12,21 +12,20 @@
...
+        batch_size = 16
+
+        y_true = torch.randint(0, n_classes, size=(n_iters * batch_size,)).to(device)
+        y_preds = torch.rand(n_iters * batch_size, n_classes).to(device)
 
         def update(engine, i):
             return (
-                y_preds[i * s + rank * offset : (i + 1) * s + rank * offset, :],
-                y_true[i * s + rank * offset : (i + 1) * s + rank * offset],
+                y_preds[i * batch_size : (i + 1) * batch_size],
```
Comment (vfdev-5) [inline tests/ignite/metrics/test_classification_report.py:27]: Still missing `:` in `y_preds`

File: tests/ignite/metrics/test_classification_report.py [hunk 4]
```diff
@@ -77,24 +80,22 @@ def update(engine, i):
...
 
         n_iters = 10
-        s = 16
+        batch_size = 16
         n_classes = 7
 
-        offset = n_iters * s
-        y_true = torch.randint(0, 2, size=(offset * idist.get_world_size(), n_classes, 6, 8)).to(device)
-        y_preds = torch.randint(0, 2, size=(offset * idist.get_world_size(), n_classes, 6, 8)).to(device)
+        y_true = torch.randint(0, 2, size=(n_iters * batch_size, n_classes, 6, 8)).to(device)
```
Comment (vfdev-5) [inline tests/ignite/metrics/test_classification_report.py:92]: nit:
```suggestion
        y_true = torch.randint(0, 2, size=(n_iters * batch_size, n_classes, 6, 8), device=device)
```

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the update, @puhuk !
One more comment to address.

**Review by vfdev-5 — APPROVED** [review]
Thanks @puhuk !

**Pull request merged**